deploy/2.mocks.ts

```diff
@@ -76,6 +76,34 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   trx = await mockCToken.setExchangeRate(BigNumber.from(10).pow(26).mul(2));
   await trx.wait();
 
+  // set initial borrow index
+  trx = await mockCToken.setBorrowIndex(BigNumber.from(10).pow(18));
+  await trx.wait();
+
+  await deploy("MockCInterestRateModel", {
+    from: deployer,
+    log: doLogging,
+  });
+
+  const mockCompoundInterestRateModel = await ethers.getContract(
+    "MockCInterestRateModel"
+  );
+
+  const secondsInYear = 31536000;
+  const secondsPerBlock = 13;
+  const ratePerBlock = BigNumber.from(10)
+    .pow(16)
+    .mul(0.02 * 100)
```

Comment:
Worth defining constants (or adding explanatory comment) for these magic values

Reply:
agreed, @ioanaroceanu2000 just to confirm this is only relevant for testing purposes of the mocks?

Reply:
yes, only relevant for testing. This is not set manually in real scenarios